annotate.el

```diff
@@ -730,6 +737,25 @@ file."
 file."
   (nth 1 record))
 
+(defun annotate-filename-from-dump (record)
+  "Get the filename field from an annotation list loaded from a
+file."
+  (cl-first record))
+
+(defun annotate-start-annotation-dump (annotation)
+  "Get the starting point of an annotation. The arg 'annotation' must be a single
+annotation field got from a file dump of all annotated buffers,
+essentially what you get from:
+(annotate-annotations-from-dump (annotate-load-annotations))). "
+  (cl-first annotation))
+
+(defun annotate-text-annotation-dump (annotation)
```

Comment:
See above: How about `annotate-text-of-annotation`?

Reply:
no problem!